### DISCUSSION TITLE
How to create area highlight annotation?

### DISCUSSION THREAD

**Discussion by chrizw** [sumatrapdfreader/sumatrapdf#3962] [Q&A]
Is there a way to create an area highlight annotation in the pdf that doesn't include any text?  not text highlight annotation. 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/16315162/86c297b2-392a-4f99-8f65-3582b184c109)

**Comment by GitHubRulesOK** [comment]
yes all highlight is an area so press the same `a`  without using the text selection (for area selection it is left-mouse CONTROL selection)

Note that highlight of text or area is down to a shape defined in MuPDF library which is curved horizontal ends (like brackets) rather than a simple rectangle, (rotation makes no difference it will be rounded each end of page horizontal axis)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/698c7b5b-a5fe-45cb-ad45-1a8ef8956061)
